js/flightlog_fields_presenter.js

```diff
@@ -343,8 +343,10 @@ function FlightLogFieldPresenter() {
                 }
 
             case 'amperageLatest':
-                if((flightLog.getSysConfig().firmwareType == FIRMWARE_TYPE_BETAFLIGHT  && semver.gte(flightLog.getSysConfig().firmwareVersion, '3.1.0')) ||
+                if((flightLog.getSysConfig().firmwareType == FIRMWARE_TYPE_BETAFLIGHT  && semver.gte(flightLog.getSysConfig().firmwareVersion, '3.1.7')) ||
                    (flightLog.getSysConfig().firmwareType == FIRMWARE_TYPE_CLEANFLIGHT && semver.gte(flightLog.getSysConfig().firmwareVersion, '2.0.0'))) {
+                       return (value / 100).toFixed(2) + "A" + ", " + (value / 100 / flightLog.getNumMotors()).toFixed(2) + "A/motor";
+                } else if(flightLog.getSysConfig().firmwareType == FIRMWARE_TYPE_BETAFLIGHT  && semver.gte(flightLog.getSysConfig().firmwareVersion, '3.1.0')) {
                     return (value / 100).toFixed(2) + "A" + ", " + (value / 100 / flightLog.getNumMotors()).toFixed(2) + "A/motor";
```

Comment:
I thought that until 3.1.7 we needed to divide by 10, and since then by 100. I've found the latest commit and seeing it I think you are right: https://github.com/betaflight/betaflight/commit/067d3c0ac2645af283f0277c78d0d2183b0156bd
I mixed the scaling factor with the amperage unit here https://github.com/betaflight/betaflight/commit/bcd5752a12f5c24bbe18db85944bca51eba08717
Good catch!

Reply:
Wasn't me, it was you who fixed it in cleanflight#61, and @wolkesson who pointed out that this had never been cherry-picked into Betaflight. Teamwork for the win!